symbology/apply-unique-values-with-alternate-symbols/src/main/java/com/esri/samples/apply_unique_values_with_alternate_symbols/ApplyUniqueValuesWithAlternateSymbolsSample.java

```diff
@@ -0,0 +1,231 @@
+
+  private Label lblScale;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Apply unique values with alternate symbols");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access base maps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a point located at San Francisco, CA to be used as the viewpoint for the map
+      Point vPoint = new Point(-13631205.660131, 4546829.846004, SpatialReferences.getWebMercator());
+
+      // create a map with the standard imagery basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // set a viewpoint to the map view centered on a point
+      mapView.setViewpointCenterAsync(vPoint, 25000.0);
+
+      // creating box, button, and label UI components
+      VBox vBox = controlsVBox();
+
+      // resetting map viewpoint on button press
+      btnReset.setOnAction(event -> {
+        // reset the map viewpoint
+        mapView.setViewpointAsync(new Viewpoint(vPoint, 25000.0), 1.5f, AnimationCurve.EASE_IN_OUT_SINE);
+      });
+
+      // updating label scale values
+      mapView.addViewpointChangedListener(event->
+        // update label text
+        lblScale.setText("Current scale: 1:" + String.format("%.2f", mapView.getMapScale())));
+
+      // creating simple markers where the blue square and yellow diamond are alternate symbols
+      // the red triangle is used to create a unique symbol and the purple diamond is the default symbol for the unique value renderer
+      SimpleMarkerSymbol sBlueSquareSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.SQUARE, 0xFF0000FF, 15);
+      SimpleMarkerSymbol sYellowDiamondSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.DIAMOND, 0xFFFFFF00, 15);
+      SimpleMarkerSymbol sRedTriangleSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, 0xFFFF0000, 15);
+      SimpleMarkerSymbol sPurpleDiamondSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.DIAMOND, 0xFF800080, 12);
+
+      // converting simpler marker symbols to multilayer point symbols
+      MultilayerPointSymbol mBlueSquareSymbol = sBlueSquareSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol mYellowDiamondSymbol = sYellowDiamondSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol mRedTriangleSymbol = sRedTriangleSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol mPurpleDiamondSymbol = sPurpleDiamondSymbol.toMultilayerSymbol();
```

Comment:
same here: let's spell it out

```suggestion
      // convert simple marker symbols to multilayer point symbols
      MultilayerPointSymbol multilayerBlueSquareSymbol = sBlueSquareSymbol.toMultilayerSymbol();
      MultilayerPointSymbol multilayerYellowDiamondSymbol = sYellowDiamondSymbol.toMultilayerSymbol();
      MultilayerPointSymbol multilayerRedTriangleSymbol = sRedTriangleSymbol.toMultilayerSymbol();
      MultilayerPointSymbol multilayerPurpleDiamondSymbol = sPurpleDiamondSymbol.toMultilayerSymbol();
```